### ISSUE TITLE
About: images lack alt text

### ISSUE THREAD

**Issue by lfrancis** [usnationalarchives/nara-bia-photos#120]
https://nara-bia-photos.herokuapp.com/about

The three images on this page need alt tex or blank alt tags.

Lizzy, is this something that needs to be done by the content owners?
